src/js/media/html5.js

```diff
@@ -307,13 +307,13 @@ vjs.Html5.nativeSourceHandler = {};
  * @return {String}         'probably', 'maybe', or '' (empty string)
  */
 vjs.Html5.nativeSourceHandler.canHandleSource = function(source){
-  var ext;
+  var match, ext;
```

Comment:
I thought the plan was to go with explicit var declarations moving forward, or are instances like this an exception (no actual values set, just defined)?